src/components/PluginCardChart.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useEffect, useRef } from 'react'
+import * as echarts from 'echarts'
+import dayjs from 'dayjs'
+import downloadIcon from '../assets/downloadIcon.svg'
+
+type InstallationData = {
+    [timestamp: string]: number
+}
+
+type DataType = {
+    installations: InstallationData
+}
+
+interface PluginCardChartProps {
+    data?: DataType
+}
+
+const PluginCardChart: React.FC<PluginCardChartProps> = ({ data }) => {
+    const chartRef = useRef(null)
+
+    useEffect(() => {
```

Comment:
Think this could be either a useMemo or a useCallback, but not a useEffect, because we are not dealing with React side effects here: https://www.w3schools.com/react/react_useeffect.asp

Reply:
This one should be followed up on too.